examples/last-10-builds-async.php

```diff
@@ -7,14 +7,14 @@
 use WyriHaximus\React\GuzzlePsr7\HttpClientAdapter;
 use WyriHaximus\Travis\BuildCollection;
 use WyriHaximus\Travis\Builds;
-use WyriHaximus\Travis\Client;
```

Comment:
@localheinz I'm refraining from merging PR's today, more tired then I thought :(. Will check it tomorrow :+1: